pkg/vm/opcodebench_test.go

```diff
@@ -74,714 +75,1578 @@ func opParamSlotsPushVM(op opcode.Opcode, param []byte, sslot int, slotloc int,
+		currDeep  = stepDeep - 1
+	)
+	for range countPoints {
+		item := stackitem.NewArray(getArrayWithDeep(currDeep))
+		testCases = append(testCases, testCase{
+			name: fmt.Sprintf("%s %d", opcode.LDSFLD0, currDeep),
+			f: func() *VM {
+				v := opVM(opcode.LDSFLD0)()
+				slot := []stackitem.Item{item}
+				v.Context().sc.static = slot
+				return v
+			},
+		})
+		currDeep += stepDeep
+	}
+	for _, tc := range testCases {
+		b.Run(tc.name, func(b *testing.B) {
+			benchOpcode(b, tc.f)
+		})
+	}
 }
 
-func BenchmarkOpcodes(t *testing.B) {
-	t.Run("NOP", func(t *testing.B) { benchOpcode(t, opVM(opcode.NOP)) })
-	t.Run("PUSHINT8", func(t *testing.B) {
-		t.Run("00", func(t *testing.B) { benchOpcode(t, opParamVM(opcode.PUSHINT8, []byte{0})) })
-		t.Run("FF", func(t *testing.B) { benchOpcode(t, opParamVM(opcode.PUSHINT8, []byte{0xff})) })
-	})
-	t.Run("PUSHINT16", func(t *testing.B) {
-		t.Run("0000", func(t *testing.B) { benchOpcode(t, opParamVM(opcode.PUSHINT16, []byte{0, 0})) })
-		t.Run("FFFF", func(t *testing.B) { benchOpcode(t, opParamVM(opcode.PUSHINT16, []byte{0xff, 0xff})) })
-	})
-	t.Run("PUSHINT32", func(t *testing.B) {
-		t.Run("0000...", func(t *testing.B) { benchOpcode(t, opParamVM(opcode.PUSHINT32, zeroSlice(4))) })
-		t.Run("FFFF...", func(t *testing.B) { benchOpcode(t, opParamVM(opcode.PUSHINT32, ffSlice(4))) })
-	})
-	t.Run("PUSHINT64", func(t *testing.B) {
-		t.Run("0000...", func(t *testing.B) { benchOpcode(t, opParamVM(opcode.PUSHINT64, zeroSlice(8))) })
-		t.Run("FFFF...", func(t *testing.B) { benchOpcode(t, opParamVM(opcode.PUSHINT64, ffSlice(8))) })
-	})
-	t.Run("PUSHINT128", func(t *testing.B) {
-		t.Run("0000...", func(t *testing.B) { benchOpcode(t, opParamVM(opcode.PUSHINT128, zeroSlice(16))) })
-		t.Run("FFFF...", func(t *testing.B) { benchOpcode(t, opParamVM(opcode.PUSHINT128, ffSlice(16))) })
-	})
-	t.Run("PUSHINT256", func(t *testing.B) {
-		t.Run("0000...", func(t *testing.B) { benchOpcode(t, opParamVM(opcode.PUSHINT256, zeroSlice(32))) })
-		t.Run("FFFF...", func(t *testing.B) { benchOpcode(t, opParamVM(opcode.PUSHINT256, ffSlice(32))) })
-	})
-	t.Run("PUSHA", func(t *testing.B) {
-		t.Run("small script", func(t *testing.B) { benchOpcode(t, opParamVM(opcode.PUSHA, zeroSlice(4))) })
-		t.Run("big script", func(t *testing.B) { benchOpcode(t, opParamVM(opcode.PUSHA, zeroSlice(4+65536))) })
-	})
-	t.Run("PUSHNULL", func(t *testing.B) { benchOpcode(t, opVM(opcode.PUSHNULL)) })
-	t.Run("PUSHDATA1", func(t *testing.B) {
-		var oneSlice = []byte{1, 0}
-		var maxSlice = zeroSlice(255)
-		maxSlice = append([]byte{255}, maxSlice...)
-
-		t.Run("1", func(t *testing.B) { benchOpcode(t, opParamVM(opcode.PUSHDATA1, oneSlice)) })
-		t.Run("255", func(t *testing.B) { benchOpcode(t, opParamVM(opcode.PUSHDATA1, maxSlice)) })
-	})
-	t.Run("PUSHDATA2", func(t *testing.B) {
-		const minLen = 256
-		const maxLen = 65535
-		var minSlice = zeroSlice(minLen + 2)
-		var maxSlice = zeroSlice(maxLen + 2)
+func BenchmarkST(b *testing.B) {
```

Comment:
Ditto here  and everywhere below. Name must contain maximum common part of benchmarked opcodes so that we can easily guess the group of opcodes from the bench name.